components/page-sections.js

```diff
@@ -12,6 +12,9 @@ import mdToHTML from './utils/md-to-html'
 const repoEditRootURL =
   'https://github.com/kentcdodds/glamorous-website/edit/master'
 
+const sectionTitleAsID = title =>
+  `${title.toLowerCase().replace(/ /g, '-').replace(/`/g, '')}`
```

Comment:
I wonder whether it'd be safer in the long-run to use [`slugify`](https://www.npmjs.com/package/slugify) instead. Wouldn't add a TON to our bundle and would probably be nicer.